shared/actions/fs/index.js

```diff
@@ -315,6 +303,31 @@ function* ignoreFavoriteSaga(action: FsGen.FavoriteIgnorePayload): Saga.SagaGene
   }
 }
 
+function* onAction(action: FsGen.OnActionPayload): Saga.SagaGenerator<any, any> {
```

Comment:
Not clear to me what `onAction` should mean.

Reply:
This is what happens when user clicks the ... button. Are you suggesting adding a comment here, or a better name?

Reply:
Better name, for all of the questions like this one.

Reply:
Would `showActionPopup` be better?

Reply:
Hmm re file: this is already in the FS saga, triggered from FsGen right? Re Saga: I'm still confused but I think this is a "task", and the whole thing is a Saga?

Reply:
Also other tasks don't have the "Saga" suffix.

Reply:
Sure, the `file` part is important because it communicates what the action is operating on. `fs.showActionPopup` doesn't really tell us anything.